ekglib/__init__.py

```diff
@@ -2,7 +2,7 @@
 """
 
 # The format of the __version__ line is matched by a regex in setup.py
-__version__ = "0.0.1"
+__version__ = "0.0.2"
```

Comment:
this shouldn't change

Reply:
is this not the version number we need to bump? you said there were two places? or you mean, we keep this static, and bump the sha, still in a base container?